[FIX] Install fails with Python 3.14

###  Description
After setting up a Python virtual environment using Python 3.14.2, scadview fails to install with an error:

>  × Failed to build `moderngl==5.12.0`
  ├─▶ The build backend returned an error
  ╰─▶ Call to `setuptools.build_meta.build_wheel` failed (exit status: 1)
  [stdout]
      running bdist_wheel
      running build
      running build_py
      copying _moderngl.py -> build/lib.macosx-14.0-arm64-cpython-314
...
      [stderr]
      /Users/neillamoureux/.cache/uv/builds-v0/.tmpSHQ8Gf/lib/python3.14/site-packages/setuptools/dist.py:765: SetuptoolsDeprecationWarning: License
      classifiers are deprecated.
      !!
...
      src/gl_methods.hpp:1318:5: error: use of undeclared identifier 'decltype'
       1318 |     load(TexParameteri);
            |     ^
      src/gl_methods.hpp:1307:36: note: expanded from macro 'load'
       1307 |     #define load(name) res.name = (decltype(res.name))load_opengl_function(loader, method, "gl" # name);
            |                                    ^
      fatal error: too many errors emitted, stopping now [-ferror-limit=]
      20 errors generated.
      error: command '/usr/bin/clang++' failed with exit code 1
      hint: This usually indicates a problem with the package or the build environment.


### Steps to reproduce

These steps use `uv`.

1.  From a terminal, create a new directory in a convenient spot.  
2. `cd` into the directory
3. `uv init --python=3.14`
4. `uv add scadview`.  The error above appears as it tries to build moderngl.
5. Click on '....'
6. Scroll down to '....'
7. See error

### Expected behavior
It should just install.

### Desktop 

- OS: e.g. macOS
- Version: Sonoma 14.7
- OpenGL version: 21.0.19

### Additional context

This failed on another Mac running Tahoe, but with an error attempting to build manifold3d.  It was complaining it needed [`tbb`](https://en.wikipedia.org/wiki/Threading_Building_Blocks) for parallel compilation.  From Wikipedia:

> Threading Building Blocks (oneTBB; formerly Threading Building Blocks or TBB) is a [C++](https://en.wikipedia.org/wiki/C%2B%2B) [template](https://en.wikipedia.org/wiki/Template_(C%2B%2B)) [library](https://en.wikipedia.org/wiki/Library_(computer_science)) developed by [Intel](https://en.wikipedia.org/wiki/Intel) for [parallel programming](https://en.wikipedia.org/wiki/Parallel_programming) on [multi-core processors](https://en.wikipedia.org/wiki/Multi-core_processor).

--- a/README.md
+++ b/README.md
@@ -28,7 +28,7 @@ Open your terminal application
 for example `cmd` in Windows
 or `Terminal.app` under Applications > Utilties > Terminal.app in macOS.
 
-In the terminal, check that you have Python 3.11 or greater via
+In the terminal, check that you have Python 3.11 through 3.13 or greater via
 
 `python --version`
 
@@ -36,7 +36,7 @@ or, if the `python` command cannot be found:
 
 `python3 --version`
 
-If Python 3.11 or greater is installed on your system, 
+If Python 3.11 through 3.13 is installed on your system, 
 you can install SCADview directly into your system.
 
 #### Virtual venv option

--- a/docs/index.md
+++ b/docs/index.md
@@ -6,6 +6,12 @@ and SCADview allows you to view your SCAD models.
 You do this by running {{ project_name }}, writing code to create a Trimesh object, 
 and loading it from the {{ project_name }} UI.
 
+## Python versions supported
+
+{{ project_name }} supports Python versions 3.11 - 3.13,
+since some packages it depends on do not supply wheels for versions > 3.13.
+Attempting to install in 3.14 or higher may result in an error.
+
 ## How it works
 
 {{ project_name }} enables a iterative work flow to build Trimesh objects.
@@ -27,7 +33,7 @@ Open your terminal application
 for example `cmd` in Windows
 or `Terminal.app` under Applications > Utilties > Terminal.app in macOS.
 
-In the terminal, check that you have Python 3.11 or greater via
+In the terminal, check that you have Python 3.11 through 3.13 via
 
 `python --version`
 
@@ -35,7 +41,7 @@ or, if the `python` command cannot be found:
 
 `python3 --version`
 
-If Python 3.11 or greater is installed on your system, 
+If Python 3.11 through 3.13 is installed on your system, 
 you can install {{ project_name }} directly into your system.
 
 #### Virtual venv option

